### ISSUE TITLE
(KNIHY) vyšlo jako...

### ISSUE THREAD

**Issue by vendaciki** [div-cz/div_app#75]
nachystala jsem ikonky na "vyšlo jako..." budeme vědět v jaké formě to všechno vyšlo

book_detail.html

![Snímek obrazovky 2024-12-26 141134](https://github.com/user-attachments/assets/825f233f-cfb4-4055-8d69-4a086fef88e1)

**Comment by xSilence8x** [comment]
@vendaciki A jaký je tvůj úkol pro nás? 😀

**Comment by vendaciki** [comment]
no, to záleží... jestli bude informace v DB v jaké formě to vyšlo, tak se ukáže ikonka. Možná to zakomponovat do editaci knihy?

**Comment by KuceraMar** [comment]
příští týden budou data co jde koupit jako ekniha nebo audio a můžeme to i přímo prodat

v knize bude uvedeno co chceme, ale asi tam dáme "Koupeno na DIV.cz" ?

Potřebujeme okno, kde je možné zakoupit.
@vendaciki je lepší samostatné? Nebo vyskakovací? Nebo to můžeme dát přímo do detailu knihy 
- formát PDF, mobi, epub
- cena 449,-
- ke stáhnutí ihned
-
